pipelines/vars/forklift_luna.yml

```diff
@@ -34,6 +34,7 @@ server_box:
         - "fb-test-foreman-ansible.bats"
         - "fb-test-foreman-templates.bats"
         - "fb-katello-proxy.bats"
+      pipeline_remove_pulp2: true
```

Comment:
Should we also run removal on proxy boxes?

Reply:
I debated it, @jlsherrill thoughts?

Reply:
probably makes sense to or else foreman-maintain will fail there too, without https://github.com/theforeman/foreman_maintain/pull/464/files

Reply:
I don't think our pipelines run foreman-maintain on the content proxies at the moment. I can run a test with it and see.